src/utils/index.ts

```diff
@@ -51,7 +51,7 @@ export const withConditional = <TProps extends {}>(
 	};
 };
 
-export const stopEvent = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+export const stopEvent = (event: any) => {
```

Comment:
Can't we do anything better?

Reply:
Don't even remember why I made this change, it's not necessary. Reverting